iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TapChangerStepAdder.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2023, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.iidm.network;
+
+/**
+ * @author Florent MILLOT {@literal <florent.millot at rte-france.com>}
+ */
+public interface TapChangerStepAdder<S extends com.powsybl.iidm.network.TapChangerStepAdder<S, A>, A extends TapChangerAdder<A, S, ?>> {
```

Comment:
To be more concise:
```suggestion
public interface TapChangerStepAdder<S extends TapChangerStepAdder<S, A>, A extends TapChangerAdder<A, S, ?>> {
```

Reply:
I did in #2829 but forgot to do it here, I commit the change